hieradata/integration.yaml

```diff
@@ -314,6 +314,7 @@ users::usernames:
   - stephenharker
   - stevelaing
   - suzannehamilton
+  - suthagarkathirkamathamby
```

Comment:
Minor comment, can this be sorted into alphabetical order?

Reply:
Hi @surminus , I have now placed my name in the correct order. Apologies for that.